### PR TITLE
fix(tag input): fix Styling

### PR DESCRIPTION
**Pull request by developerstew**
## Description

Brief description of changes made e.g. bug fix for broken Ajax container.

## Related tickets

Any Jira tickets related to the changes.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

- [ ] I have lint my changes locally.
- [ ] I have formatted my changes locally.
- [ ] I have updated any relevant documentation.
- [ ] I have added tests to cover my changes.
- [ ] I have checked that all new and existing tests passed.
- [ ] I have checked to ensure there aren't other open Pull Requests for the same update/change.

### CODE REVIEW

File: src/components/detail-layout/index.stories.js [hunk 1]
```diff
@@ -17,7 +20,26 @@ Primary.args = {
...
-	children: () => <div />,
+	children: () => (
+		<Formik
+			initialValues={{
+				heading: '',
+				name: 'Example'
+			}}
+		>
+			{({values, handleChange}) => (
+				<FieldSet id="basicInfo" title="This is not working">
```
Comment (mantonides) [inline src/components/detail-layout/index.stories.js]: I hope it's working now

File: src/components/detail-topbar/index.js [hunk 2]
```diff
@@ -35,36 +35,33 @@ export const DetailTopbar = ({
...
-								type="submit"
-								isLoading={isSubmitting}
-								onClick={handleConfirm}
-							>
-								Save
+			{isDirty || value.includes('_Copy') ? (
+				<ActionCluster className={styles.ButtonContainer}>
+					<ConfirmDialog
+						id="confirm-dialog"
+						title={`Save  ${name}?`}
```
Comment (mantonides) [inline src/components/detail-topbar/index.js:42]: Just make sure of your spacing please

**Review by mantonides — APPROVED** [review]

File: src/components/detail-topbar/index.js (hunk 2 above)
Comment (developerstew) [reply src/components/detail-topbar/index.js:42]: Sure thing

File: src/components/detail-layout/index.js [hunk 3]
```diff
@@ -49,7 +56,11 @@ export const DetailLayout = ({
 										classes.content
 									)}
 								>
-									<form className={cn(styles.Form)} onSubmit={handleSubmit}>
+									<form
+										className={cn(styles.Form)}
+										onChange={handleChange}
```
Comment (tiaanduplessis) [inline src/components/detail-layout/index.js:61]: I don't think the `handleChange `event is required on the form element. It is handled at input level.

**Review by tiaanduplessis — CHANGES_REQUESTED** [review]

File: src/components/detail-layout/index.js (hunk 3 above)
Comment (developerstew) [reply src/components/detail-layout/index.js:61]: @tiaanduplessis. This was actually what fixed the 'dirty' prop from formik that was not working correctly. I did try removing this.

Comment (tiaanduplessis) [reply src/components/detail-layout/index.js:61]: Strange. I'll need to look into it at later stage, but we can merge for now to unblock you

**Pull request merged**